src/targets/commitOnGitRepository.ts

```diff
@@ -134,7 +135,18 @@ export async function pushArchiveToGitRepository({
       const git = simpleGit(directory);
 
       logger?.info(`Cloning ${repositoryUrl} into ${directory}...`);
-      await git.clone(repositoryUrl, directory);
+
+      // We can assume that the provided url is a valid URL (otherwise parsing will crash)
+      const parsedUrl = new URL(repositoryUrl);
```

Comment:
l: I'm fine with letting things crash but let's maybe add an error message saying that only https urls will work, wdyt?

Reply:
Makes sense! b3becf94f1eeb977956aedf517167af8086b853e